assets/stylesheets/_components.scss

```diff
@@ -6,9 +6,9 @@
 // stylesheet order does not matter.
 // ==========================================================================
 
-@import 'components/button/style';
+@import 'woocommerce/components/button/style';
 @import 'components/button-group/style';
-@import 'components/card/style';
+@import 'woocommerce/components/card/style';
```

Comment:
I noticed I had overlooked to change this import when replacing the Card component.